cmd/gapit/flags.go

```diff
@@ -256,4 +256,22 @@ type (
 			Count int `help:"number of frames after Start to process: -1 for all frames"`
 		}
 	}
+	TrimFlags struct {
+		Gapis    GapisFlags
+		Gapir    GapirFlags
+		Commands bool `help:"Treat every command as its own frame"`
+		No       struct {
+			DCE bool `help:"Prevent dead code elimination"`
+			New struct {
+				State bool `help:"Prevent the construction of a new initial state"`
```

Comment:
It would be good to document somewhere (even if its just a comment) what this actually means.
What is the end-user impact of creating an initial state.
Is it the case that NOT specifying this flag will result in incorrect traces being generated?
If the last part is true, I would rather the correct trace be generated in the initial case, and a "smaller but possibly broken" trace being generated when specifying a flag.

Reply:
`no-new-state` is only relevant with with the InitialState rebuilding code, which is currently broken and not included in this PR. I will also remove this flag from this PR.

Once we have InitialState rebuilding, the user is not likely to care about this flag, but it is useful for testing/debugging.